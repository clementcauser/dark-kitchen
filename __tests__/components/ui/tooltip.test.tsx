import {
  TooltipProvider,
  Tooltip,
  TooltipTrigger,
  TooltipContent,
} from "@radix-ui/react-tooltip";
import { render } from "@testing-library/react";

describe("tooltip", () => {
  it("renders correctly", () => {
    const { container } = render(
      <TooltipProvider>
        <Tooltip>
          <TooltipTrigger asChild>
            <h1>Hover me</h1>
          </TooltipTrigger>
          <TooltipContent>
            <p>Add to library</p>
          </TooltipContent>
        </Tooltip>
      </TooltipProvider>
    );

    expect(container).toMatchSnapshot();
  });
});
