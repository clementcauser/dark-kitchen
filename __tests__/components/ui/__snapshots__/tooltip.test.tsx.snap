// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tooltip renders correctly 1`] = `
<div>
  <h1
    data-state="closed"
  >
    Hover me
  </h1>
</div>
`;
